tools/update-dll-msvc.bat

```diff
@@ -0,0 +1,193 @@
+rem \     'mac' : 'make -f make_mac.mak',
+rem \     'linux' : 'make',
+rem \     'unix' : 'gmake',
+rem \    },
+rem \ }
+
+setlocal enabledelayedexpansion
+
+rem detect the cpu architecture is 32 bit or 64 bit processor
+reg query "HKLM\Hardware\Description\System\CentralProcessor\0" /v "Identifier" | find /i "x86" > NUL && set cpu_arch=i386 || set cpu_arch=AMD64
+
+rem --------------------------------------------------------------------------------------------
+rem  See https://docs.microsoft.com/zh-cn/archive/blogs/david.wang/howto-detect-process-bitness
+rem  In an ideal world, everything runs as native bitness (64bit program on 64bit OS, 
+rem  32bit program on 32bit OS) and life goes on. However, sometimes you need to run 
+rem  that legacy 32bit program on a 64bit OS and need to configure things a little 
+rem  differently as a result. How can you detect this WOW64 case (32bit program on 64bit OS) 
+rem  and act appropriately?
+rem
+rem  Detection Matrix
+rem  The general idea is to check the following environment variables:
+REM
+rem  PROCESSOR_ARCHITECTURE - reports the native processor architecture EXCEPT for WOW64, where it reports x86.
+rem  PROCESSOR_ARCHITEW6432 - not used EXCEPT for WOW64, where it reports the original native processor architecture.
+rem
+rem  Visually, it looks like:
+REM
+rem  Environment Variable\Program Bitness	32bit Native	64bit Native	WOW64
+rem  PROCESSOR_ARCHITECTURE	                x86	            AMD64	        x86
+rem  PROCESSOR_ARCHITEW6432	                undefined	    undefined	    AMD64
+rem
+rem  WOW64 = 32bit Program on 64bit OS
+rem --------------------------------------------------------------------------------------------
+set vimproc_arch=64
+set msvc_arch=x86_amd64
+if "%PROCESSOR_ARCHITECTURE%"=="x86" (if not defined PROCESSOR_ARCHITEW6432 set "vimproc_arch=32" & set "msvc_arch=x86")
+
+set vimproc_dllname=vimproc_win%vimproc_arch%.dll
+
+rem --------------------------------------------------------------------------------------------
+rem Determine which registry keys to look at based on architecture type.
+rem --------------------------------------------------------------------------------------------
+if "%vimproc_arch%"=="64" (
+    set VCRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\SxS\VC7
+    set VSRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\VisualStudio\SxS\VS7
+    rem Visual Studio 2010 Express edition need to work with Windows SDK 7.1 if the msvc_arch is x86_amd64
+    set WinSDKRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\WOW6432Node\Microsoft\Microsoft SDKs\Windows\v7.1
+)    else (
+    set VCRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\SxS\VC7
+    set VSRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\Wow6432Node\Microsoft\VisualStudio\SxS\VS7
+    rem Visual Studio 2010 Express edition need to work with Windows SDK 7.1 if the msvc_arch is x86_amd64
+    set WinSDKRegKeyPath=HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v7.1
```

Comment:
Looks wrong.
You specify `/reg:32` option to `reg` command, then always Wow6432Node is used. Adding Wow6432Node doubles it.

Reply:
@k-takata ,  To specify /reg:32 is work no matter at WOW6432Node or not. I will change like this:
![image](https://user-images.githubusercontent.com/20179047/72693726-18b1c900-3b6d-11ea-802a-6115481757c8.png)

Reply:
@k-takata , after remove to support VC2010, I simple the registry keys as below:
![image](https://user-images.githubusercontent.com/20179047/72696068-de005e80-3b75-11ea-8221-88ca13b969e1.png)

Reply:
I think Wow6432Node is not needed at all when `/reg:32` is specified.